src/components/game/CamButton.tsx

```diff
@@ -16,10 +20,19 @@ const CamButton = () => {
     emitUpdateUserStream({ video: changedCamState });
   };
   return (
-    <button onClick={handleClick} aria-label={`캠 ${userCam ? '켜짐' : '꺼짐'}`}>
-      {userCam ? 'CAM켜짐' : 'CAM꺼짐'}
-    </button>
+    <CAMButton onClick={handleClick} aria-label={`캠 ${userCam ? '켜짐' : '꺼짐'}`}>
+      {userCam ? 'CAM켜짐' : <img src={camIcon} />}
+    </CAMButton>
   );
 };
 
+// TODO: CAM 커짐/켜짐 아이콘 바꾸기
+
+const CAMButton = styled.button`
```

Comment:
```suggestion
const CamButtonLayout = styled.button`
```

괜찮습니다 이렇게 쓰죠 파스칼케이스로

Reply:
@pds0309 컨벤션대로면 button인데 layout을 붙여도 되나여??

Reply:
![image](https://user-images.githubusercontent.com/76927397/212541164-a4dc60ee-82d1-4e07-a2e5-fa8b501a98b3.png)